### PR TITLE
GA Version - 2020-01-01 for version - 2019-10-01-preview

### PR DESCRIPTION
**Pull request by navysingla**
This GA version 2020-01-01 is the exact copy of version 2019-10-01-preview

### CODE REVIEW

File: specification/subscription/resource-manager/Microsoft.Subscription/stable/2020-01-01/subscriptions.json [hunk 1]
```diff
@@ -0,0 +1,640 @@
...
+      }
+    }
+  },
+  "parameters": {
+    "apiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "Version of the API to be used with the client request. Current version is 2019-10-01-preview"
```
Comment (MyronFanQiu) [inline specification/subscription/resource-manager/Microsoft.Subscription/stable/2020-01-01/subscriptions.json]: should it be 2020-01-01?

**Review by mmyyrroonn — APPROVED** [review]
LGTM

File: specification/subscription/resource-manager/Microsoft.Subscription/stable/2020-01-01/subscriptions.json (hunk 1 above)
Comment (navysingla) [reply specification/subscription/resource-manager/Microsoft.Subscription/stable/2020-01-01/subscriptions.json]: Yes, fixed

**Pull request merged**